docs/man-openmpi/man3/MPI_Aint_add.3.rst

```diff
@@ -68,7 +68,7 @@ sum of the *base* and *disp* arguments, where *base* represents a base
 address returned by a call to :ref:`MPI_Get_address` and *disp* represents
 a signed integer displacement. The resulting address is valid only at
 the process that generated *base*, and it must correspond to a location
-in the same object referenced by *base*, as described in MPI-3.1 section
+in the same object referenced by *base*, as described in MPI-|mpi_standard_version|.|mpi_standard_subversion| section
```

Comment:
Do you know that the MPI spec section number in question is the same in all versions of the MPI spec?  Sometimes the section numbers change between different versions of the MPI standard.

(same goes for all other similar locations that you changed like this)